src/test/java/io/github/classgraph/issues/issue305/Issue305Test.java

```diff
@@ -0,0 +1,90 @@
+package io.github.classgraph.issues.issue305;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import io.github.classgraph.ClassGraph;
+import io.github.classgraph.ScanResult;
+
+/**
+ * Issue305.
+ */
+public class Issue305Test {
+    private ConsoleHandler errPrintStreamHandler = null;
+    private final Logger rootLogger = Logger.getLogger("");
+
+    /** Reset encapsulation circumvention method after each test. */
```

Comment:
Comment is incorrect (came across via copy/paste from other test)

Reply:
Hmm, for clarification, I don't think I added this comment. I think it was already there.